File: JSLife/plant.js
const _DRAW_DATA_PLANT = new DrawData(
  SQUARE,
  '#00ff00',
  false
);

class Plant extends Entity {
  constructor(x = 0, y = 0) {
    super(x, y);
  }
  
  draw() {
    return _DRAW_DATA_PLANT;
  }
}

File: Python/PyLife/plant.py
from draw_handler import DrawData, GREEN
from entity import Entity
from grid_point import GridPoint
from polygon import SQUARE


class Plant(Entity):
    DRAW_DATA = DrawData(
        SQUARE,
        GREEN,
        False
    )

    def __init__(self, position: GridPoint):
        super().__init__(position)

    def draw(self) -> DrawData:
        return Plant.DRAW_DATA
